### PR TITLE
Update Manual Build Instructions - node 7.5 to 8.7

### PR DESCRIPTION
**Pull request by TimCliff**

### CODE REVIEW

**Comment by bnchdrff** [comment]
what versions of npm & yarn are used in the docker image?

**Comment by bnchdrff** [comment]
(i think we might need yarn 1.3.x at this point)

**Comment by relativityboy** [comment]
@TimCliff - is there an issue open for this that we can point to?

File: README.md [hunk 1]
```diff
@@ -47,13 +47,13 @@ mkdir tmp
 
 #### Install dependencies
 
-Install at least Node v7.5 if you don't already have it. We recommend using `nvm` to do this as it's both the simplest way to install and manage installed version(s) of node. If you need `nvm`, you can get it at [https://github.com/creationix/nvm](https://github.com/creationix/nvm).
+Install at least Node v8.7 if you don't already have it. We recommend using `nvm` to do this as it's both the simplest way to install and manage installed version(s) of node. If you need `nvm`, you can get it at [https://github.com/creationix/nvm](https://github.com/creationix/nvm).
 
-Condenser is known to successfully build using node 7.5, npm 4.1.2, and yarn 1.1.0.
+Condenser is known to successfully build using node 8.7, npm 4.1.2, and yarn 1.1.0.
```
Comment (relativityboy) [inline README.md]: @TimCliff - Could you update this to say `yarn 1.3.2` ?

**Comment by TimCliff** [comment]
@bnchdrff @relativityboy I updated yarn to 1.3.2. Should I also update npm to 5.4.2 based on https://github.com/steemit/condenser/blob/master/package.json ?

Also, there is no issue for this. I just submitted the PR when I noticed that the build instructions wern't working anymore. Should I be creating issues for all PRs?

**Comment by TimCliff** [comment]
I went ahead and made the change to update npm to 5.4.2, since I'm fairly certain that is right. Let me know if it is not, and I'll roll it back.

**Comment by bnchdrff** [comment]
npm is tied to node really, so it's not super relevant anyways -- that is my mistake for including it in that blurb in README. this is good to go.

**Pull request merged**